frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -60,7 +60,9 @@ export function Surveys(): JSX.Element {
 
     const { user } = useValues(userLogic)
     const { currentTeam } = useValues(teamLogic)
-    const surveysPopupDisabled = currentTeam && !currentTeam?.surveys_opt_in
+    const { updateCurrentTeam } = useActions(teamLogic)
+    const surveysPopupDisabled =
```

Comment:
I'd prob rename this `showSurveysDisabledBanner` - I was looking around for a popup window but it's just a banner. Might also be better to put this in surveysLogic selectors but I go the short&lazy route sometimes too haha